src/FakeItEasy/A.cs

```diff
@@ -156,5 +156,17 @@ public static IPropertySetterAnyValueConfiguration<TValue> CallToSet<TValue>(Exp
         {
             return ConfigurationManager.CallToSet(propertySpecification);
         }
+
+        /// <summary>
+        /// Configures subscription to or unsubscription from an event of a faked object.
+        /// </summary>
+        /// <param name="fake">The fake to configure.</param>
+        /// <param name="descriptor">An <see cref="EventAction"/> that represents the action to configure.</param>
+        /// <returns>A configuration object.</returns>
+        public static IAnyCallConfigurationWithVoidReturnType CallTo(object fake, EventAction descriptor)
+        {
+            Guard.AgainstNull(descriptor, nameof(descriptor));
+            return CallTo(fake).WithVoidReturnType().Where(descriptor.Matches, writer => descriptor.WriteDescription(fake, writer));

```

Comment:
The methods in `A` usually defer to `ConfigurationManager`, but since this can be expressed as a call to `A.CallTo(fake)`, I thought it wasn't worth the trouble...

Reply:
I'm all for using our building blocks to help us.